framework/platform/bpl/uci/scripts/platform_daemon_ugw.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/sh
+ap_to_repeater() 
+{
+    #configure APM
+    #uci set beerocks.config.management_mode="Proprietary-Mesh"
+    ubus call uci set '{ "config" : "beerocks" ,"section" : "config", "values" : {"management_mode" : "Proprietary-Mesh"}}'
+    #uci set beerocks.config.operating_mode="WDS-Repeater"
+    ubus call uci set '{ "config" : "beerocks" ,"section" : "config", "values" : {"operating_mode" : "WDS-Repeater"}}'
+    #uci set beerocks.config.ssid='adam_wave_43'
+    ubus call uci set '{ "config" : "beerocks" ,"section" : "config", "values" : {"ssid" : "adam_GW_2.4G"}}'
+    #uci set beerocks.config.mode_enabled='None'
+    ubus call uci set '{ "config" : "beerocks" ,"section" : "config", "values" : {"mode_enabled" : "None"}}'
```

Comment:
Better make a single ubus call that sets all values:

```
ubus call uci set '{ "config" : "beerocks" ,"section" : "config", 
    "values" : {
        "management_mode" : "Proprietary-Mesh",
        "operating_mode" : "WDS-Repeater",
        "ssid" : "adam_GW_2.4G",
        "mode_enabled" : "None"
    }}'
```

Same below, of course.

Reply:
Why not simply use `uci set X=Y`???

Reply:
> Why not simply use `uci set X=Y`???

In UGW there is great emphasis on user permissions and ownership.
when you perform the uci set+commit  not via ubus you will change the ownership of the wireless/beerocks config file you are editing.
In the wireless uci configuration for example - this can prevent the hostapd from accessing the configuration file.
the owner of all configuration files uner /etc/config/ in UGW is rpcd and I followed the same line for beerocks

there is a discussion regarding UGW bringup and how exactly should this script and prplMesh/beerocks run and with which privileges - but decision wasn't made yet

Reply:
OK. Anyway, the whole thing needs to be updated according to https://git.prpl.dev/prplmesh/iwlwav/blob/iwlwav-19.07/prplmesh/files/etc/config/prplmesh
As you can see the config name is no longer `beerocks`, it is `prplmesh`.

In addition, I think we should rename this file to `platform_intel.sh` which will be sourced by `prplmesh_utils.sh` and used if the platform is Intel (UGW or RDKB).

Reply:
I agree - but note there are 2 files for platform_daemon, one for RDKB (already available) and one for UGW (the new file introduced). we can check if unifying them to one file is beneficial - and how much they overlap or not as a future improvement

Reply:
changed the UCI set to one call for all params